### DISCUSSION TITLE
Edit Engagement Report Template

### DISCUSSION THREAD

**Discussion by tandhy** [DefectDojo/django-DefectDojo#5914] [Q&A] [answered]
Hi team,

I've noticed that the instant report generated is using this template - engagement_pdf_report.html under dojo/templates/dojo folder on uwsgi_1 instance.
I use docker for the defect dojo so I turned the docker down, update the above HTML file and fired it up again.
I thought I could see any changes but apparently the DefectDojo did not take the changes.
Am I missing anything?

**Comment by mtesauro** [comment, marked as answer]
Docker images don't retain changes after being restarted.  You'll either need to build your own docker image with the changes you want or follow a process like outlined at https://stackoverflow.com/questions/41067032/how-to-stop-relaunch-docker-container-without-losing-the-changes
